environment.yml

```diff
@@ -19,6 +20,4 @@ dependencies:
   - tensorboardX>=2.0
   - torchvision>=0.5.0
   - tqdm>=4.45.0
-  - pip:
-    - typed-argument-parser>=1.6.1
-    - git+https://github.com/bp-kelley/descriptastorus
+  - descriptastorus
```

Comment:
Would you please add a comment about why "astartes" is not in the environment.yml file? I see that this dependency is provided in the pyproject.toml. However, there are some overlaps between these two files. We probably wanna clean up the dependencies and make then consistent. Thanks.

Reply:
I have updated the environment file to match the pyproject. The only difference is `astartes[molecules]` is now `astartes` and `aimsim` (which `astartes` uses to deal with `molecules`) between the two.